models/aeif_cond_beta_multisynapse.h

```diff
@@ -454,9 +500,38 @@ aeif_cond_beta_multisynapse::set_status( const DictionaryDatum& d )
   // consistent.
   Archiving_Node::set_status( d );
 
+  /**
+   * Here is where we must update the recordablesMap_ if new receptors
+   * are added!
+   */
+  DynamicRecordablesMap< aeif_cond_beta_multisynapse > rtmp =
+    recordablesMap_;                         // temporary copy in case of errors
+  if ( ptmp.E_rev.size() > P_.E_rev.size() ) // Number of receptors increased
+  {
+    for ( size_t receptor = P_.E_rev.size(); receptor < ptmp.E_rev.size();
+          ++receptor )
+    {
+      size_t elem = aeif_cond_beta_multisynapse::State_::G
+        + receptor * aeif_cond_beta_multisynapse::State_::
+                       NUMBER_OF_STATES_ELEMENTS_PER_RECEPTOR;
+      rtmp.insert( get_g_receptor_name( receptor ),
+        aeif_cond_beta_multisynapse::DataAccessFunctor( this, elem ) );
+    }
+  }
+  else if ( ptmp.E_rev.size()
+    < P_.E_rev.size() ) // Number of receptors decreased
```

Comment:
It would be better to have the comparison on one line and the comment on another.

Reply:
Done in eb41277